dwave/cloud/api/client.py

```diff
@@ -131,9 +133,108 @@ def request(self, method: str, *args, **kwargs):
         return response
 
 
-class VersionedAPISession(LoggingSession):
+class PayloadCompressingSession(LoggingSession):
+    """A :class:`requests.Session` subclass (technically, a further specialized
+    :class:`.LoggingSession`) that adds support for payload compression on the
```

Comment:
These two classes do not create links because Requests is not part of our intersphinx map. Consider making links to the docs for these in the `requests.Session <https://requests.readthedocs.io/en/latest/api/#requests.Session>`_ format

Reply:
Note that the whole `dwave.cloud.api.*` subspace is not added to docs yet (it's considered private), but point taken - I'll add Requests to the intersphinx map.

Reply:
Okay, I added it (and ``urllib3``) to the **SDK's** intersphinx map, where it will actually do something useful😁.
(In the unification PR.)

Reply:
Thanks! I was planning to do that prior to releasing 8.2.